src/Mobs/Ocelot.cpp

```diff
@@ -0,0 +1,194 @@
+
+#include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
+
+#include "Ocelot.h"
+#include "../World.h"
+#include "../Entities/Player.h"
+#include "../Items/ItemHandler.h"
+#include "Broadcaster.h"
+
+
+
+
+
+cOcelot::cOcelot(void) :
+	super("Ocelot", mtOcelot, "entity.cat.hurt", "entity.cat.death", 0.6, 0.8),
+	m_IsSitting(false),
+	m_IsTame(false),
+	m_IsBegging(false),
+	m_Type(0),
+	m_OwnerName("")
+{
+}
+
+
+
+
+
+void cOcelot::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
+{
+	super::Tick(a_Dt, a_Chunk);
+	if (!IsTicking())
+	{
+		// The base class tick destroyed us
+		return;
+	}
+
+	if (!IsTame() && !IsBaby())
+	{
+		cPlayer * a_Closest_Player = m_World->FindClosestPlayer(GetPosition(), 10);
```

Comment:
Normally the ocelot should run away from people (I haven't implement that) so if 2 players are standing close to each other it shouldn't really matter because it will run away from the player without the breeding items. (I think)